Backend/api/views/building.py

```diff
@@ -3,22 +3,55 @@
 from webargs import fields
 from webargs.flaskparser import use_args
 # se poate folosi si marshmallow ca se se creeze o schema pentru args
-from models import Building
+from models import Building, Connector, Group, Waypoint, StudiesIn
 import neomodel
 
 
+def test():
+    A = Group(name='A3', buildingName='Building0').save()
+    B = Waypoint.nodes.get_or_none(name='testRoom3')
+    C = A.classes.connect(B, {'course': 'BD', 'dayOfWeek': 'Monday', 'startTime': '12:00', 'finishTime': '14:00'})
+    pass
+
+
 class BuildingView(FlaskView):
     base_args = ['args']
-    excluded_methods = []  # metode care nu corespund unei rute
+    excluded_methods = [test]  # metode care nu corespund unei rute
 
     # findAll
     def index(self):
         return jsonify(list(map(lambda building: building.__properties__, Building.nodes.all()))), 200
 
-    def get(self, uid):  # trebuie schimbat sa returneze toate datele despre o cladire/ eventual cautand dupa nume
-        building = Building.nodes.get_or_none(uid=uid)
+    def get(self, name):  # trebuie schimbat sa returneze toate datele despre o cladire/ eventual cautand dupa nume
+        building = Building.nodes.get_or_none(name=name)
+        map = {"name": building.name, "uid": building.uid, "floors": [], "connectors": []}
         if building:
-            return jsonify(building.__properties__)
+            for floor in building.floors:
+                map["floors"].append({"level": floor.level, "waypoints": []})
+                for nodes in floor.waypoints:
+                    map["floors"][-1]["waypoints"].append(
+                        {"uid": nodes.uid, "floorLevel": nodes.floorLevel, "name": nodes.name,
+                         "type": nodes.labels()[1], "neighbors": []})
+                    if nodes.labels()[1] == "ClassRoom":
+                        map["floors"][-1]["waypoints"][-1]["schedule"] = []
+                        for cclass in Group.nodes.has(classes=True):
```

Comment:
[group for group in Group.nodes.has(classes=True) if group.classes.relationship(waypoint)]
Cred ca ar merge. E mai ok sa filtram sa fie doar grupele care au ora la cel waypoint decat sa facem loop peste toate si sa verificam